streams/src/main/java/org/apache/kafka/streams/kstream/internals/StatefulProcessorNode.java

```diff
@@ -28,40 +26,38 @@
 class StatefulProcessorNode<K, V> extends StatelessProcessorNode<K, V> {
 
     private final String[] storeNames;
-    private final org.apache.kafka.streams.processor.StateStoreSupplier<KeyValueStore> storeSupplier;
     private final StoreBuilder<KeyValueStore<K, V>> storeBuilder;
+    private final String maybeRepartitionedSourceName;
 
 
-    StatefulProcessorNode(final String parentNodeName,
-                          final String processorNodeName,
-                          final ProcessorSupplier<K, V> processorSupplier,
+    StatefulProcessorNode(final String nodeName,
+                          final ProcessorParameters processorParameters,
                           final String[] storeNames,
-                          final org.apache.kafka.streams.processor.StateStoreSupplier<KeyValueStore> storeSupplier,
+                          final String maybeRepartitionedSourceName,
```

Comment:
Line 37 below: it seems like the `storeBuilder` here is only for the Materialized KTable itself; if yes could we make it explicit by renaming?